redfish-core/src/error_messages.cpp

```diff
@@ -2173,6 +2173,32 @@ void mutualExclusiveProperties(crow::Response& res, const std::string& arg1,
     addMessageToErrorJson(res.jsonValue, mutualExclusiveProperties(arg1, arg2));
 }
 
+/**
+ * @internal
+ * @brief Formats RestrictedRole into JSON
+ *
+ * See header file for more information
+ * @endinternal
+ */
+nlohmann::json restrictedRole(const std::string& arg1)
```

Comment:
Why can't we just pull in a new base registry and use it from there? https://redfish.dmtf.org/registries/

Reply:
Will handle this later.  :-)